Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -52,6 +52,12 @@ public struct TuistCommand: AsyncParsableCommand {
                         ShareCommand.self,
                     ]
                 ),
+                CommandGroup(
+                    name: "AI",
```

Comment:
I wonder if we should have an "Insights" group instead and have `tuist inspect` there, too, instead of creating a group which has just one command right now. But not too opinionated about this one.